mrmustard/lab_dev/circuit_components.py

```diff
@@ -95,6 +98,46 @@ def __init__(
                 if self._representation:
                     self._representation = self._representation.reorder(tuple(perm))
 
+    def serialize(self) -> Tuple[Dict[str, Any], List[Tuple[str, ArrayLike]]]:
```

Comment:
We could avoid typing imports if we just used `tuple`, `dict`, `any`, etc

Reply:
tbh I wasn't sure when to use the `typing` aliases, but I see that a lot of it is historical. I'll remove the ones that don't add much value